### PR TITLE
Create branch feature/project-labels

### PR DESCRIPTION
**Pull request by AndryBasaraba**

### CODE REVIEW

File: components/TaskCard/index.js [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
       </b>
       {visible && (
         <Tasks>
-          {project.tasks.map((task, index) => (
+          {tasks.map((task, index) => (
             <Task key={index}>{task}</Task>
           ))}
         </Tasks>
       )}
+      <Labels>
```
Comment (jonas8900) [inline components/TaskCard/index.js:25]: the name "Labels" let me think that we are in a form right now. Maybe you could change the name to "TopicHeading" or "SingleHeadline".

File: components/TaskCard/index.js [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
 
 export const TaskCard = ({ project, index }) => {
   const [visible, setVisible] = useState(false);
   const projectClickHandler = () => {
     setVisible(!visible);
   };
+  console.log(project, index);
+  const { projectTitle, tasks, projectDue, projectSize } = project;
   return (
     <TaskListItem onClick={projectClickHandler}>
```
Comment (jonas8900) [inline components/TaskCard/index.js:13]: This is a very good idea! Maybe there could be an info that you can expand the card? Maybe with: "click for more information about your project". if we start from the dumbest users i could imagine that the info would be helpful.

**Review by jonas8900 — COMMENTED** [review]
the rest looks good to me !

File: components/TaskCard/index.js (hunk 2 above)
Comment (AndryBasaraba) [reply components/TaskCard/index.js:13]: Thank you, well noticed. I will do it in the next US.

File: components/TaskCard/index.js (hunk 1 above)
Comment (AndryBasaraba) [reply components/TaskCard/index.js:25]: It's just the name as I see it. in general I am glad that there are no errors. Thanks, it's always nice to hear opinions.

Comment (jonas8900) [reply components/TaskCard/index.js:25]: haha i feel you !

**Comment by jonas8900** [comment]
lgtm ! but the preview for vercel show multiple links, maybe you can fix this before you go in the QA

**Comment by AndryBasaraba** [comment]
I have a problem with Versel so far I can't solve it on my own I'll talk to the teacher tomorrow

**Pull request merged**